flow360/component/utils.py

```diff
@@ -48,6 +50,31 @@ def wrapper_func(*args, **kwargs):
     return wrapper
 
 
+def confirm_proceed():
```

Comment:
rename function:
`shared_account_confirm_proceed()` or something similar to indicate that this confirmation is only in case of shared account

Reply:
Renamed to shared_account_confirm_submit() in 644f8ee